### DISCUSSION TITLE
split queryParam

### DISCUSSION THREAD

**Discussion by ckapitzke** [mockoon/mockoon#565] [Help] [answered]
Hello,
is it possible to split a queryParam directly?

i have this url part:
`find/?articleNumber=3624885,36248850,36248850,362488504`

but if i try follwing:
`{{#each (split (queryParam 'articleNumber') ',')}}
  itemthis,
{{/each}}`

the result ist empty.

Best regards,
Christian

**Comment by 255kb** [comment, marked as answer]
`queryParam` has a special behavior, and returns a SafeString behind the scene which makes it incompatible with `split`. I will open an issue to fix this. 
However, we added the helper `queryParamRaw` in the latest version and it should work. 
I also realize that there is a mistake in the documentation, `this` should be inside double curly braces `{{this}}`:
```
{{#each (split (queryParamRaw 'articleNumber') ',')}} item{{this}}, {{/each}}
```
Thank you for catching this issues :)

**Reply by ckapitzke** [reply]
Great, this solves my problem :-) :+1: 

Thank you so much
